cli/project/files.go

```diff
@@ -130,6 +139,26 @@ func GetStateboardEntrypointPath(ctx *context.Ctx) string {
 	return filepath.Join(ctx.Running.AppDir, ctx.Running.StateboardEntrypoint)
 }
 
+func getFlag(conf map[string]interface{}, opts FlagOpts) (bool, error) {
+	var flag bool
+
+	if value, found := conf[opts.ConfSectionName]; found {
+		var ok bool
+		if flag, ok = value.(bool); !ok {
+			return false, fmt.Errorf("Stateboard value should be `true` or `false`")
+		}
+
+		if opts.SpecifiedFlag != flag && flag == false {
+			flag = opts.SpecifiedFlag
+			log.Warnf("Value of `stateboard` parameter in .cartridge.yml is ignored, because passed --stateboard flag")
```

Comment:
Generally it's not a warning - values from  `.cartridge.yml` are default values, that can be overwritten by passing flags on running comand.